tests/graph/benches/manual_queries/entity_queries/mod.rs

```diff
@@ -247,6 +247,31 @@ fn format_traversal_params(params: &SubgraphTraversalParams) -> String {
                 .collect::<Vec<_>>()
                 .join(";")
         ),
+        SubgraphTraversalParams::Mixed {
+            entity_traversal_paths,
+            ontology_graph_resolve_depths: depths,
+        } => format!(
+            "traversal_paths={},resolve_depths=inherit:{};values:{};properties:{};links:{};\
+             link_dests:{};;type:{}",
+            entity_traversal_paths
+                .iter()
+                .map(|path| format!(
+                    "({})",
+                    path.edges
+                        .iter()
+                        .map(|edge| format!("{edge:?}"))
+                        .collect::<Vec<_>>()
+                        .join(";")
```

Comment:
nit: you no longer need to `collect` -> `join`, you can use [intersperse](https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method.intersperse) with `Cow` + `collect` to achieve the same without any intermediary allocation.


```suggestion
                    path.edges
                        .iter()
                        .map(|edge| Cow::Owned(format!("{edge:?}")))
                        .intersperse(Cow::Borrowed(";"))
                        .collect::<String>()
```